docs/sources/integrations/servicenow/index.md

```diff
@@ -21,130 +21,105 @@ weight: 500
 
-```text
-https://<INSTANCE>.service-now.com/api/now/table/incident/{{responses.<WEBHOOK_ID>.result.sys_id}}
-```
-
-Replace `<INSTANCE>` with your ServiceNow instance, and `<WEBHOOK_ID>` with the ID of the [webhook used for creating incidents](#create-incidents-in-servicenow).
-
-- Username: Username of the [ServiceNow user](#prerequisites)
-
-- Password: Password of the [ServiceNow user](#prerequisites)
-
-Use the following JSON template as webhook data:
-
-```json
-{
-  "assigned_to": "{{user.email}}"
-}
-```
-
->**Note**: The incident will be assigned to the user that acknowledged the alert group in Grafana OnCall.
-The assignment will fail if the user email does not exist in ServiceNow.
-
-## Resolve incidents in ServiceNow
-
-The steps below describe how to create an outgoing webhook in Grafana OnCall that will allow to automatically close
-incidents in ServiceNow when an alert group is resolved in Grafana OnCall.
-
-- Trigger type: `Resolved`
-
-- Integrations: Select integrations that will trigger the webhook
-
-- HTTP method: `PUT`
-
-- Webhook URL:
-
-```text
-https://<INSTANCE>.service-now.com/api/now/table/incident/{{responses.<WEBHOOK_ID>.result.sys_id}}
-```
-
-Replace `<INSTANCE>` with your ServiceNow instance, and `<WEBHOOK_ID>` with the ID of the [webhook used for creating incidents](#create-incidents-in-servicenow).
-
-- Username: Username of the [ServiceNow user](#prerequisites)
+## Prerequisites
 
-- Password: Password of the [ServiceNow user](#prerequisites)
+1. Create a new ServiceNow user to be used by Grafana OnCall. On your ServiceNow instance,
+navigate to **User Administration** > **Users** and click **New**. Fill in the following details:
+   * Username: `grafana-oncall`
+   * First name: `Grafana OnCall`
+   * Active: ✔
+   * Web service access only: ✔
+
+   After creating the user, generate and save a password using the Set Password button for later use.
+2. Grant the following roles to the user (use the **Roles** tab):
+   * `itil` (allows creating and updating incidents)
+   * `personalize_choices` (allows fetching the list of available incident states)
+
+## Create integration
+
+1. On the **Integrations** tab, click **+ New integration**.
+2. Select **ServiceNow** from the list of available integrations.
+3. Enter a name and description for the integration.
+4. Enter ServiceNow credentials (instance URL, username, and password) and verify the connection.
+5. Make sure **Create default outgoing webhooks** is enabled. This will create the necessary webhooks in Grafana OnCall
+to send alerts to ServiceNow.
+6. Click **Create integration**.
+7. Map ServiceNow incident states to OnCall alert group statuses. Example:
+     * `Firing -> New`
+     * `Acknowledged -> In Progress`
+     * `Resolved -> Resolved`
+     * `Silenced -> Not Selected`
+8. Generate a ServiceNow Business Rule script. This script will allow your ServiceNow instance to send updates to
```

Comment:
Wondering if we should make it more explicit that there is a token being generated and the script will be displayed to copy only once? Otherwise a regenerate action will be needed (new token, new script)

Reply:
good call! https://github.com/grafana/oncall/pull/4215/commits/ce0961d46d21b58a5189e82c566e27d6f7799681